locales/sv/zodiac.json

```diff
@@ -0,0 +1,62 @@
+{
+	"aries": {
+		"name": "Väduren",
+		"element": 0,
+		"stone": "blodsten"
+	},
+	"taurus": {
+		"name": "Oxen",
+		"element": 1,
+		"stone": "safir"
+	},
+	"gemini": {
+		"name": "Tvillingarna",
+		"element": 2,
+		"stone": "agat"
+	},
+	"cancer": {
+		"name": "Kräftan",
+		"element": 3,
+		"stone": "emerald"
```

Comment:
In Swidish, is it "emerald" or "smaragd"?

Reply:
My bad, emerald is actually the english word. Correct swedish should be "Smaragd"